modules/sdk-coin-atom/src/lib/transactionBuilder.ts

```diff
@@ -153,14 +155,14 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
   private validateAmountData(amountArray: Coin[]): void {
     // TODO - make the error messages more descriptive of which amount data is failing
     // Do amount checking by casting to numbers/bignumber and comparing as well
-    for (const amount of amountArray) {
-      if (!amount.amount) {
-        throw new BuildTransactionError('Invalid amount: undefined');
-      }
-      if (!amount.denom) {
-        throw new BuildTransactionError('Invalid denom: undefined');
-      }
-    }
+    // for (const amount of amountArray) {
+    //   if (!amount.amount) {
+    //     throw new BuildTransactionError('Invalid amount: undefined');
+    //   }
+    //   if (!amount.denom) {
+    //     throw new BuildTransactionError('Invalid denom: undefined');
+    //   }
+    // }
```

Comment:
Please remove the commented code and add the ticket number in todo.

Reply:
Marking as resolved